pkgs/applications/networking/instant-messengers/beeper/default.nix

```diff
@@ -44,6 +54,20 @@ mkDerivation rec {
     runHook postInstall
   '';
 
+  passthru = {
+    updateScript = lib.getExe (writeShellApplication {
+      name = "update-beeper";
+      runtimeInputs = [ curl yq common-updater-scripts ];
+      text = ''
+        set -o errexit
+        latestLinux="$(curl -s https://download.todesktop.com/2003241lzgn20jd/latest-linux.yml)"
+        version="$(echo "$latestLinux" | yq -r .version)"
+        filename="$(echo "$latestLinux" | yq -r '.files[] | .url | select(. | endswith(".AppImage"))')"
+        update-source-version beeper "$version" "" "https://download.todesktop.com/2003241lzgn20jd/$filename" --source-key=src.src
+      '';
+    });
+  };
```

Comment:
Really clean. Thanks for introducing this to me! I haven't been the best at keeping up on a lot of this so I appreciate you contributing the `updateScript` so I could learn about it. Ill have to give it a try for the other package I maintain.

Reply:
```
Going to be running update for following packages:
 - beeper-3.80.17

Press Enter key to continue...

Running update for:
 - beeper-3.80.17: UPDATING ...
 - beeper-3.80.17: DONE.

Packages updated!
```